ckanext/geocat/utils/csw_mapping.py

```diff
@@ -374,7 +383,7 @@ def _map_dataset_see_alsos(node, organization_slug, valid_identifiers):
     return []
 
 
-def _map_dataset_rights(node, terms_of_use):
+def _map_dataset_rights(node, terms_of_use, default_rights):
     GMD_RIGHTS = './/gmd:resourceConstraints//gmd:otherConstraints'
```

Comment:
There are some other constants that are defined in functions as well that I didn't spot earlier (like this one) - we should move them up to the head of the file so that they don't get redefined every time the functions are called.